cardano-node/src/Cardano/Node/StateRep.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
Can we move this into the `Tracing` folder?

Reply:
@Jimbo4350 Done.